public/index.html

```diff
@@ -4,6 +4,37 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
+    <script src="%PUBLIC_URL%/outdated-browser-rework.min.js"></script>
+    <script>
+      outdatedBrowserRework({
+        browserSupport: {
+          Chrome: 67, // Includes Chrome for mobile devices
+          Edge: 40,
+          Safari: 10,
+          "Mobile Safari": 10,
+          Opera: 54,
+          Firefox: 60,
+          Vivaldi: 1,
+          IE: false
+        },
+        isUnknownBrowserOK: true,
+        messages: {
+          en: {
+            outOfDate: "This browser may be too old for Terra to function correctly",
```

Comment:
If a user sees this message, they are able to click "Close" to try to use Terra anyways, is that correct? I wanted to confirm the UX flow with the message. If so, looks good!

Reply:
I'd like to indicate the action the user should take to fix the issue. Also, is there a reason for "too old"? It sounds a little informal. What about:
"This browser may be out of date, and Terra may not function correctly. Please upgrade your browser version."

Unless the reason for "too old" is an accessibility concern. I know sometimes more common words are preferable for ESL users. But out of date feels common enough in computer land? We're dealing with a science-professional audience.

Reply:
Is it always a case of "too old"? What about browsers like Safari that we don't support any version of? We've had 'blank-screen' issues there. An upgrade would not fix that. Can we distinguish these cases?

If so, the fully-unsupported browser message might be:
"Terra may not function correctly on this browser. If you experience issues, please try using Google Chrome."